qiskit_ibm_runtime/transpiler/passes/basis/fold_rzz_angle.py

```diff
@@ -244,3 +248,163 @@ def _quad4(angle: float, qubits: Tuple[Qubit, ...]) -> DAGCircuit:
             check=False,
         )
         return new_dag
+
+
+def convert_to_rzz_valid_pub(
```

Comment:
Since this is not a transpiler pass that will be run automatically, do you want to expose it in a more public way by including it in the documentation and possibly providing a higher level import path than `qiskit_ibm_runtime.transpiler.passes.basis.fold_rzz_angle`?

Reply:
@ElePT 
1. Which import level do you think fits?
2. Where does the relevant documentation reside?